pylabrobot/resources/resource_stack.py

```diff
@@ -95,26 +96,42 @@ def get_actual_resource_height(resource: Resource) -> float:
       return max(get_actual_resource_height(child) for child in self.children)
     return sum(get_actual_resource_height(child) for child in self.children)
 
-  def assign_child_resource(self, resource: Resource, location: Optional[Coordinate] = None,
-    reassign: bool = False):
+
+  def get_resource_stack_size(self) -> Coordinate:
     if self.direction == "x":
       resource_location = Coordinate(self.get_size_x(), 0, 0)
```

Comment:
potentially just put this in get_default_child_location. It seems incorrect to have a get_size method return a coordinate

Reply:
it's used in the assign lid to resource stack special case. I can rename to something if you don't like the `size`  word.

Reply:
for this particular function i don't like it :)